src/containers/Pomodoro/Pomodoro.js

```diff
@@ -20,16 +20,18 @@ const Button = styled.button`
   border-style: none;
 `;
 
-const SESSION_LENGTH = 5;
-const BREAK_LENGTH = 3;
+const SESSION_LENGTH = 10;
+let BREAK_LENGTH = 3;
```

Comment:
All-caps-snake-case naming is by convention reserved for constants in most programming languages. Leave it as const and and rename to `SHORT_BREAK_LENGTH` and add `LONG_BREAK_LENGTH`.